pkg/java/jar.go

```diff
@@ -76,6 +77,57 @@ func ReadMethodByteCode(jarFile string, className string) (bytecode [][]byte, er
 	return ExtractBytecode(buf.Bytes())
 }
 
+type AverageJavaNameSizes struct {
+	PackageName float32
+	ClassName   float32
+}
+
+// AveragePackageAndClassLength produces an average of the package and class name lengths.
+// In the event that the given Jar file contents are incredibly large, and either the total
+// number of classes or packages, or the sums of their lengths, exceeds a unit32 then the
+// result will be inaccurate.
```

Comment:
Is it known how it will be inaccurate? Will those extra contents just not be considered as part of the average, or something else? Think it's worth clarifying.